src/SourceBuild/content/test/tests.proj

```diff
@@ -20,7 +20,11 @@
     <_RunScenarioTests Condition="'$(DotNetBuildPass)' != '' and '$(DotNetBuildPass)' != '1'">false</_RunScenarioTests>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TestInstallerPackages)' == 'true'">
+    <ProjectReference Condition="'$(TestRpmPackages)' == 'true' or '$(TestDebPackages)' == 'true'" Include="Microsoft.DotNet.Installer.Tests\Microsoft.DotNet.Installer.Tests.csproj" BuildInParallel="false" />
+  </ItemGroup>
```

Comment:
Looks like when `TestInstallerPackages` is true, nothing else from this project should get invoked. Did you consider adding a `--projects` switch to build.sh that allows to pass in the project to build directly to overrides tests.proj?

Such a switch would have helped me locally as well.

Reply:
Thanks, that seems like a better option. We rarely group tests together in a single instance - explicitly running specific test projects seem like the best option.

It makes sense to allow `--projects` switch to override the projects we build, in general. I.e. we'd allow override of `build.proj` as well, and not just the `tests.proj`

Reply:
It seems that it is easy to allow override with a single project, provided on command line, but support for many projects didn't work when I tried to implement it (i.e. with `/p:Projects=`).

I've searched the whole .NET codebase. Arcade and various repos allow `--projects` but in all use cases, a single project is provided as input.

Reply:
Yes the flag should only support a single project but that should be OK. I.e. you can introduce a new Traversal.proj file, i.e. `test\installer-tests.proj` (similar to the `test\tests.proj` one) that is used as the entry-point test project.

Reply:
OK. For now we won't need the traversal project. I'll add the project override in a moment.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/47313/commits/156990979ed61bf851c3bdc057f1069e714c0620

I've added a new switch `--projects`. Single project can be specified directly, multiple projects can be added via traversal project. Even though this switch accepts a single project on cmd-line I've kept the same switch name as in all other repos. Plural in the name is still true if specifying a traversal project. Perhaps I should clarify this a bit in the switch comment.